### PR TITLE
[FR] Italics osu! in Rules

### PR DESCRIPTION
**Pull request by RedNesto**

### CODE REVIEW

File: wiki/Rules/fr.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ Règles du Chat En-Jeu
...
 7.  **Gardez les liens pertinents et pointant vers des sites légaux.** Les sites de streaming (ceux ayant une licence comme Animelab et Crunchyroll sont OK) et tout ce qui est lié au "warez" ou aux logiciels piratés de quelque sorte qu'ils soient ne sont pas tolérés, et nous ne les voulons pas dans notre chat. Partager des liens Twitch pour d'autres jeux est toléré, mais SEULEMENT dans `#videogames`.
-8. **Ne partagez pas de serveurs de chat tierces, ce qui inclut les groupes/serveurs Discord, Teamspeak et Skype.** Le chat d'osu! est à propos d'osu! et ne sert pas à partager les liens de vos serveurs personnels. Vous pouvez le faire par message personnel si vous en avez vraiment besoin.
+8. **Ne partagez pas de serveurs de chat tierces, ce qui inclut les groupes/serveurs Discord, Teamspeak et Skype.** Le chat d'osu! est à propos d'_osu!_ et ne sert pas à partager les liens de vos serveurs personnels. Vous pouvez le faire par message personnel si vous en avez vraiment besoin.
```
Comment (TPGPL) [inline wiki/Rules/fr.md]: `Le chat d'osu!` missed

File: wiki/Rules/fr.md [hunk 2]
```diff
@@ -22,7 +22,7 @@ Règles du Chat En-Jeu
...
 9.  **Dans les canaux spécifiques aux langues, vous devez parler la langue dominante (Anglais pour #english, Thaï pour #thai, etc) si vous participez à une discussion.** Quand la langue d'un canal n'est pas spécifiée (ou sous-entendue par le nom du canal, comme Thaï pour `#thai` ou Néerlandais pour `#dutch`), c'est toujours l'Anglais qui est utilisé. Les canaux `#lobby` et `#help` peuvent être utilisés avec n'importe quelle langue, à l'instar des autres canaux. *Les membres de l'[équipe de modération](/wiki/People/Global_Moderation_Team) n'ont pas besoin de suivre cette règle lorsqu'ils font leur travail.*
 
 Règles du Forum
```
Comment (TPGPL) [inline wiki/Rules/fr.md]: missed many of them; use Find function

Comment (RedNesto) [inline wiki/Rules/fr.md]: I just don't know if things like `osu! community` should be italicized

Comment (SpartanPlume) [inline wiki/Rules/fr.md]: yeah, I was thinking this too. But should they be translated then ?

Comment (RedNesto) [inline wiki/Rules/fr.md]: Loctav translated them in #465

Comment (TPGPL) [inline wiki/Rules/fr.md]: Just make sure every reference to game or client *itself* is italic

**Pull request merged**